File: laramie-seguridad-webapi/src/domain/entities/verificacion.ts
export default class Verificacion {

    id: number;
	idTipoVerificacion: number;
	idEstadoVerificacion: number;
	idUsuario: number;
	codigo: string;
	fechaDesde: Date;
	fechaHasta: Date;
	token: string;
	detalle: string;

	constructor(
        id: number = 0,
		idTipoVerificacion: number = 0,
		idEstadoVerificacion: number = 0,
		idUsuario: number = 0,
		codigo: string = "",
		fechaDesde: Date = null,
		fechaHasta: Date = null,
		token: string = "",
		detalle: string = ""
	)
	{
        this.id = id;
		this.id = id;
		this.idTipoVerificacion = idTipoVerificacion;
		this.idEstadoVerificacion = idEstadoVerificacion;
		this.idUsuario = idUsuario;
		this.codigo = codigo;
		this.fechaDesde = fechaDesde;
		this.fechaHasta = fechaHasta;
		this.token = token;
		this.detalle = detalle;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.id = row.id ?? 0;
		this.idTipoVerificacion = row.idTipoVerificacion ?? 0;
		this.idEstadoVerificacion = row.idEstadoVerificacion ?? 0;
		this.idUsuario = row.idUsuario ?? 0;
		this.codigo = row.codigo ?? "";
		this.fechaDesde = row.fechaDesde ?? null;
		this.fechaHasta = row.fechaHasta ?? null;
		this.token = row.token ?? "";
		this.detalle = row.detalle ?? "";
	}

}

File: modules/seguridad/models/verificacion.py
from sqlalchemy import Column, BigInteger, String, DateTime, ForeignKey
from sqlalchemy.sql import func

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Verificacion(Base):
    __tablename__ = "seguridad_verificaciones"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    id_tipo_verificacion = Column(BigInteger, nullable=False)
    id_estado_verificacion = Column(BigInteger, nullable=False, default=51)
    id_usuario = Column(BigInteger, ForeignKey("seguridad_usuarios.id"), nullable=True)
    codigo = Column(String(50), nullable=False)
    fecha_desde = Column(DateTime(timezone=True), server_default=func.now(), nullable=False)
    fecha_hasta = Column(DateTime(timezone=True), nullable=False)
    token = Column(String(250), nullable=False, unique=True, index=True)
    detalle = Column(String(1000), nullable=False, default="")
